sass/_header.scss

```diff
@@ -143,76 +132,76 @@ Nav menu, which animates into close x
 --------------------------------------------------------------*/
 
 .hamburger {
-	border: 0;
-    display: inline-block;
-	height: 30px;
-    margin: 1px auto -15px;
-    padding-top: 5px;
-    -webkit-tap-highlight-color: transparent;
-    width: 28px;
-    
-    span {
-        display: block;
-        position: relative;
-        
-        &:before,
-        &:after {
-            content: '';
-            left: 0;
-            position: absolute;
-            top: -9px;
-        }
-        
-        &:after {
-            top: 9px;
-        }
+  border: 0;
+  display: inline-block;
+  height: 30px;
+  margin: 1px auto -15px;
+  padding-top: 5px;
+  -webkit-tap-highlight-color: transparent;
+  width: 28px;
+
+  span {
+    display: block;
+    position: relative;
+
+    &:before,
+    &:after {
+      content: '';
+      left: 0;
+      position: absolute;
+      top: -9px;
+    }
+
+    &:after {
+      top: 9px;
     }
+  }
 }
 
 .hamburger span,
 .hamburger span:after,
 .hamburger span:before {
-	backface-visibility: hidden;
-	background-color: #fff;
-	border-radius: 2px;
-	height: 5px;
-    transition: all .5s;
-	width: 100%;
+  backface-visibility: hidden;
+  background-color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.